game/resource/English/ability/items/tooltip_greater_arcane_boots.txt

```diff
@@ -3,13 +3,14 @@
 //=================================================================================
 "DOTA_Tooltip_Ability_item_greater_arcane_boots"                          "Greater Arcane Boots"
 "DOTA_Tooltip_Ability_item_recipe_greater_arcane_boots"                   "Greater Arcane Boots Recipe"
-"DOTA_Tooltip_ability_item_greater_arcane_boots_Description"              "<h1>Active: Replenish Mana</h1>Restores %replenish_amount% mana to all nearby allies.<br><br>Radius: %replenish_radius%\n<h1>Passive: Greed</h1> Generates unreliable gold.\n#{fools_gold_penalty_warning}\nFlat movement speed bonuses from multiple pairs of boots do not stack.<br><font color='#FFA500'>Can be upgraded to Greater Guardian Greaves of the same level by combining with level 1 Mekansm and the regular Guardian Greaves Recipe.</font>"
+"DOTA_Tooltip_ability_item_greater_arcane_boots_Description"              "<h1>Active: Replenish Mana</h1>Restores %replenish_amount% mana to all nearby allies.<br><br>Radius: %replenish_radius%\n<h1>Passive: Greed</h1> Generates unreliable gold.\n#{fools_gold_penalty_warning}\n<h1>Passive: Inflation</h1>Passive gold income is modified by an amount that starts at 100% and ticks down by 1% every 4 seconds to a minimum of 30%. Can be reset by picking up a bounty rune. <br><br>Maximum Percentage: %gold_given_max_percentage%%<br>Minimum Percentage: %gold_given_min_percentage%%<br>Tick Interval: %tick_interval% Seconds\nFlat movement speed bonuses from multiple pairs of boots do not stack.<br><font color='#FFA500'>Can be upgraded to Greater Guardian Greaves at level 4 by combining with level 1 Mekansm and the regular Guardian Greaves Recipe, but will lose the passive gold income.</font>"
```

Comment:
I think something like: "Passive gold from Greed reduces by 1% every 4 seconds to a minimum of 30%. Can be reset by picking up a bounty rune." is a better description. Should also reference the KV. So like "... to a minimum of %gold_given_min_percentage%...".

Reply:
Ya I just had that as a placeholder. Will change.